website/server/libs/applePayments.js

```diff
@@ -142,13 +142,13 @@ api.subscribe = async function subscribe (sku, user, receipt, headers, nextPayme
 
 
 api.cancelSubscribe = async function cancelSubscribe (user, headers) {
-  let data = user.purchased.plan.additionalData;
+  let plan = user.purchased.plan;
 
-  if (!data) throw new NotAuthorized(shared.i18n.t('missingSubscription'));
+  if (plan.paymentMethod !== 'Apple') throw new NotAuthorized(shared.i18n.t('missingSubscription'));
```

Comment:
should we use the constan `applePayments.constants.PAYMENT_METHOD_APPLE ` since it exists?